examples/website/collision-filter/app.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {createRoot} from 'react-dom/client';
+import {Map} from 'react-map-gl';
+import maplibregl from 'maplibre-gl';
+import DeckGL from '@deck.gl/react';
+import {GeoJsonLayer, TextLayer} from '@deck.gl/layers';
+import data from './data/road_accidents.json';
```

Comment:
We need to remember to change data sources to [deck.gl-data](https://github.com/visgl/deck.gl-data)

Reply:
Added in this PR: https://github.com/visgl/deck.gl-data/pull/31